app/enterprise/2.3.x/admin-api/licenses/reference.md

```diff
@@ -5,22 +5,22 @@ book: licenses
 licenses_attribute_id: |
     Attributes | Description
     ---:| ---
-    `id` | The **License's** unique ID.
+    `id` | The **license's** unique ID.
 
 licenses_body: |
     Attribute | Description
     ---:| ---
-    `payload` | The **{{site.ee_product_name}} License** in JSON format.
+    `payload` | The **Kong Gateway license** in JSON format.
```

Comment:
Should this be `{{site.ee_base_gateway}}` or `{{site.base_gateway}}`?

Reply:
It breaks when you try to use it in yaml front matter, so unfortunately it has to be plaintext here. Like so: https://docs.konghq.com/enterprise/2.3.x/admin-api/licenses/reference/#request-body